### ISSUE TITLE
ZeroDivisionError while updating progress

### ISSUE THREAD

**Issue by jordyamc** [DevilXD/TwitchDropsMiner#493]
I got this error, first time it shows up

```
10:31:03: Fatal error encountered:
10:31:03: 
10:31:03: Traceback (most recent call last):
10:31:03:   File "main.py", line 158, in main
10:31:03:   File "twitch.py", line 765, in run
10:31:03:   File "twitch.py", line 810, in _run
10:31:03:   File "twitch.py", line 1695, in fetch_inventory
10:31:03:   File "gui.py", line 1371, in add_campaign
10:31:03:   File "gui.py", line 1413, in update_progress
10:31:03:   File "functools.py", line 981, in __get__
10:31:03:   File "inventory.py", line 219, in progress
10:31:03: ZeroDivisionError: division by zero
```

**Comment by biast12** [comment]
getting the same error

**Comment by DevilXD** [comment]
Hello.

The miner isn't functional due to #462. If you are using the fork, please report this issue under the fork's repository instead.

If you are using this version, then it won't work due to #462. This error (if really present) will be fixed during #462 resolve.

I'm leaving this up in case anyone runs into the same thing, to try and avoid duplicate issues.

**Comment by DevilXD** [comment]
This should now be fixed via https://github.com/DevilXD/TwitchDropsMiner/commit/c7f8d56946c7ef4cf1bbd42e0aa7cf48332b2000 in the latest master build.
